spec/requests/api/v1/customers/applied_coupons_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::Customers::AppliedCouponsController, type: :request do
```

Comment:
RSpec/FilePath: Spec path should end with `api/v1/customers/applied_coupons_controller*_spec.rb`. (https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/FilePath)